### DISCUSSION TITLE
How can I instruct evans to display the default values?

### DISCUSSION THREAD

**Discussion by ghevge** [ktr0731/evans#403] [Q&A]
I am calling my grpc endpoint using evans. All good, except for the situations when my response contains enum type fields and their value is set to the default value ( index = 0). In this situation, evans doesn't print out that field at all.
How can I instruct evans to display the default values?

Is there any "emiDefaults" equivalent flag available in evans, as grpccurl has ? https://github.com/fullstorydev/grpcurl/issues/95

Thanks

**Comment by ktr0731** [comment]
There are no ways to display default values. So it should be supported as a new feature. 
Can you create a new issue?

**Comment by ghevge** [comment]
Created https://github.com/ktr0731/evans/issues/404. Thanks
